src/components/pages/Login/_SignupPageStyled.less

```diff
@@ -72,8 +73,8 @@ button.buttonStyles {
   padding: 1.5rem;
   &:hover {
     background-color: #ffffff;
-    color: #bd5a27;
-    border: 1px solid #bd5a27;
+    color: @button;
+    border: 1px solid @button;
```

Comment:
Love the reuse of LESS variables, something I love to see and need to remember to do more personally!

Reply:
Agreed! Nice reuse, and clean up of the styling code here!